bftengine/src/bftengine/ReplicaConfigSerializer.hpp

```diff
@@ -0,0 +1,61 @@
+//Concord
+//
+//Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+//
+//This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+//This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "ReplicaConfig.hpp"
+#include "Serializable.h"
+
+namespace bftEngine {
+namespace impl {
+
+// As ReplicaConfig is an interface struct, we need this additional wrapper
```

Comment:
Wouldn't it be better to add serialize/deserialize methods to ReplicaConfig, so that we wouldn't have to keep the classes in sync when separated across the project?

Reply:
See the comment.